lib/index.js

```diff
@@ -388,6 +389,24 @@ MapboxGeocoder.prototype = {
     return this.options.proximity;
   },
 
+  /**
+   * Get the text to use as the search bar placeholder
+   * 
+   * If placeholder is provided in options, then use options.placeholder
+   * Otherwise, if language is provided in options, then use the localized string of the first language if available
+   * Otherwise use the default
+   */
+  getPlaceholderText: function(){
```

Comment:
should this method be private?

Reply:
related to https://github.com/mapbox/mapbox-gl-geocoder/pull/198/files#r265396617 but I think it's better to re-gen the docs with each commit and include them.

Reply:
Good catch. Moved this to be a private method.